pkg/registry/certificates/rest/storage_certificates.go

```diff
@@ -60,6 +70,24 @@ func (p RESTStorageProvider) v1Storage(apiResourceConfigSource serverstorage.API
 	return storage, nil
 }
 
+func (p RESTStorageProvider) v1alpha1Storage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) (map[string]rest.Storage, error) {
+	storage := map[string]rest.Storage{}
+
+	if apiResourceConfigSource.ResourceEnabled(certificatesapiv1alpha1.SchemeGroupVersion.WithResource("clustertrustbundles")) {
```

Comment:
use the:

```
if resource := "clustertrustbundles"; ...
  storage[resource] = ...
```

pattern to avoid setting up copy/paste landmines

Reply:
Done.